dash-pipeline/SAI/specs/dash_tunnel.yaml

```diff
@@ -49,6 +49,32 @@ sai_apis:
     valid_only: null
     is_vlan: false
     deprecated: false
+  - !!python/object:utils.sai_spec.sai_attribute.SaiAttribute
```

Comment:
Can we please make the SAI_DASH_TUNNEL_ATTR_DASH_ENCAP and SAI_DASH_TUNNEL_ATTR_TUNNEL_KEY create-only as well ?

Reply:
@r12f , can you check?

Reply:
Tunnel s/be updatable?  Unlikely that we will update a VNI inside of a tunnel, however IPs inside may change (create-set).  (Key=VNI).
Would we have a case where we change the key (MSEE for example)?  Yes, we have seen this before.  Can be handled as tunnel replacement on the route.  

@r12f and @mzms to finalize - s/be ok to be create-only

Reply:
I think this can be create_only. I don't see a case where we want to update VNI on the fly. If its required, we can create new tunnel with new members, bind and then remove the old tunnel.

Reply:
Yes, since the DASH TUNNEL is now a group of tunnels, modifying encap/VNI on all tunnels in the group is non-trivial.

IP change should be fine since my understanding is that any IP change in the northbound DASH_TUNNEL will convert to delete old DASH_TUNNEL_MEMBER and then add new DASH_TUNNEL_MEMBER at the SAI API level. This is fine as long as the total members stays within the max member size configured initially.

Reply:
Updated now.